terraform/github.py

```diff
@@ -76,6 +76,7 @@ def __init__(self, pr_url: str):
                     self._plan = match.group(1).strip()
                     self._status = match.group(2).strip()
                     return
+        raise Exception(f'No matching plan as comment on PR {pr_url} by username {github_username}.')
```

Comment:
Added this as currently if you set a github email as the USERNAME env, plan works okay but apply steps fail with just: 
```
None

Exited with code exit status 1
```

Reply:
I have a feeling this would break the apply step, as the comment won't exist yet

Reply:
```suggestion

```